go.mod

```diff
@@ -4,22 +4,22 @@ go 1.19
 
 require (
 	github.com/briandowns/spinner v1.19.0
-	github.com/finleap-connect/monoskope v0.4.15
+	github.com/finleap-connect/monoskope v0.5.2-0.20221027115350-9fd1a29a5af2
```

Comment:
if you are planning to release we should update this.

Reply:
agreed